index.html

```diff
@@ -0,0 +1,122 @@
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Art Exhibition: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./img/log.jpg" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-md-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="body-bg d-flex justify-content-center">
+      <div class="header">
+        <div class="title d-flex justify-content-center flex-column">
+          <h4>"Hello! Sharing world"</h4>
+          <h1>ART EXHIBITION - 2022</h1>
+          <div class="descruption">
+            <p>Art exhibitions represent the gathering of art objects into a space for a temporary event. The exhibition or show may include work by a single artist, art works on a single topic or theme, from a specific era, in a defined medium, from a defined geographic region, or a combination of all of these.</p>
+          </div>
+          <div class="prod-year">
+            <h3>
+              2022.08.15 (Monday) ~ 16(Tuesday)
+            </h3>
+            <p>@informations by Mufungizi Gloire</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <div class="container-programs">
+    <div class="main-program-container container-fluid text-light programs-class">
+      <h1 class="text-center my-2">Main Programs</h1>
+      <div class="container d-flex my-3 justify-content-center line-bellow"></div>
```

Comment:
- [ ] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> You're already using `h1` tag here ```  <h1>ART EXHIBITION - 2022</h1> ```, and that's good because this is the main title 💯😉